reservations/views.py

```diff
@@ -1,6 +1,83 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.contrib.auth import login
+from django.contrib.auth.decorators import login_required
+
+from . import forms
+from .forms import ReservationForm, StaffReservationForm
+
+from accounts.models import UserProfile, User
 
 
 def dashboard(request):
     """Dashboard page for the staff reservation website"""
     return render(request, "reservations/index.html", {"title": "Home"})
+
+
+def staff_sign_up_for_guest(request):

```

Comment:
```suggestion
def add_reservation(request):

```
Let's rename this view so it makes sense.